libraries/Adapters/Microsoft.Bot.Builder.Adapters.Webex/WebexHelper.cs

```diff
@@ -130,26 +127,27 @@ public static Activity DecryptedMessageToActivity(Message decryptedMessage, Pers
             {
                 // strip the mention & HTML from the message
                 var pattern = new Regex($"^(<p>)?<spark-mention .*?data-object-id=\"{identity.Id}\".*?>.*?</spark-mention>", RegexOptions.IgnoreCase | RegexOptions.Multiline);
-                if (!decryptedMessage.Html.Equals(pattern))
+                if (!pattern.IsMatch(decryptedMessage.Html))
                 {
                     // this should look like ciscospark://us/PEOPLE/<id string>
-                    var match = Regex.Match(identity.Id, "/ciscospark://.*/(.*)", RegexOptions.IgnoreCase | RegexOptions.Multiline);
+                    var match = Regex.Match(identity.Id, "ciscospark://.*/(.*)", RegexOptions.IgnoreCase | RegexOptions.Multiline);
```

Comment:
Same on the regex thing.

Reply:
This is the one that still needs review. But I wanted to send to MS what we have so far so Gabo can see the progress.

Reply:
OK, send these changes, but don't forget to review this with me later.